server/utils/viewUtils.ts

```diff
@@ -95,4 +96,22 @@ export default class ViewUtils {
       items,
     }
   }
+
+  static linkHtml(links: { text: string; href: string; hiddenText?: string }[]): string {
```

Comment:
Do you have a test for this?

Reply:
Good spot! No, I was under the impression that we hadn't tested any of `ViewUtils`. That's not quite true - we have a test for a handful of its methods, but not for most. I'll add some on Monday.